pages/scrolls.xml

```diff
@@ -16,17 +16,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <sourceDesc>
                 <p>encodedd in TEI</p>
             </sourceDesc>
-        </fileDesc>       
-        <revisionDesc>       
-            <change who="PL" when="2020-06-22">Created page</change>     
+        </fileDesc>
+        <revisionDesc>
+            <change who="PL" when="2020-06-22">Created page</change>
+            <change who="DE" when="2021-02-10">Added the examples on encoding of physical description.</change>    
         </revisionDesc>
     </teiHeader>
     <text>
         <body xml:id="scrolls">
             <div>
+              <head>Description of Scrolls</head>
                 <p>We treat Scrolls in the same way as we do with <ref target="manuscripts">Manuscripts</ref> and any other written artefacts,
```

Comment:
I know it is an old sentence, but let's take the opportunity to make it correct:
"We treat scrolls in the same way as `<ref target="manuscripts">codices</ref>`"